templates/rhpam73-managed.yaml

```diff
@@ -80,40 +80,40 @@ parameters:
   example: "maven1!"
   required: false
 - displayName: KIE Admin User
-  description: KIE administrator username
+  description: KIE administrator username.
   name: KIE_ADMIN_USER
   value: adminUser
   required: false
 - displayName: KIE Admin Password
-  description: KIE administrator password
+  description: KIE administrator password.
   name: KIE_ADMIN_PWD
   from: "[a-zA-Z]{6}[0-9]{1}!"
   generate: expression
   required: false
 - displayName: KIE Server User
-  description: KIE server username (Sets the org.kie.server.user system property)
+  description: KIE server username. (Sets the org.kie.server.user system property)
   name: KIE_SERVER_USER
   value: executionUser
   required: false
 - displayName: KIE Server Password
-  description: KIE server password (Sets the org.kie.server.pwd system property)
+  description: KIE server password. (Sets the org.kie.server.pwd system property)
   name: KIE_SERVER_PWD
   from: "[a-zA-Z]{6}[0-9]{1}!"
   generate: expression
   required: false
 ## OpenShift Enhancement BEGIN
 - displayName: Prefer KIE Server OpenShift Service [Tech Preview]
-  description: Enables connection to KIE Server via OpenShift internal Service endpoint (Sets the org.kie.server.controller.openshift.prefer.kieserver.service system property)
+  description: If OpenShift Integration of the Business Central is turned on, set this parameter to true to enables connection to KIE Server via an OpenShift internal Service endpoint. Otherwise the Business Central uses a Route to connect to KIE Server. (Sets the org.kie.server.controller.openshift.prefer.kieserver.service system property)
```

Comment:
I do not believe "Otherwise the Business Central uses a Route to connect to KIE Server." is correct. @rhtevan , please suggest a better description?